src/deadline/job_attachments/api/attachment.py

```diff
@@ -145,12 +147,13 @@ def attachment_upload(
             job_attachment_settings=s3_settings,
             manifest=manifest,
             partial_manifest_prefix=upload_manifest_path,
+            manifest_name_suffix=manifest_name_suffix,
             source_root=Path(rule.source_path),
             asset_root=Path(rule.destination_path),
             s3_check_cache_dir=config_file.get_cache_directory(),
         )
         logger.echo(
-            f"Uploaded assets from {rule.source_path}, to {s3_settings.to_s3_root_uri()}/{key}, hashed data {data}"
+            f"Uploaded assets from {rule.destination_path}, to {s3_settings.to_s3_root_uri()}/Manifests/{key}, hashed data {data}"
```

Comment:
Is changing the message to `rule.destination_path` intentional? Aren't we uploading from `source_path`?

Reply:
The `source_path` is the submission path on user's workstation, the `destination_path` is the mapped path on worker, hence the destination path should be the correct path.

Reply:
Right, but I feel like this would be awkward since the log happens on attachment upload. Is this code shared between both submitter and worker flows? (uploading inputs and uploading outputs)?

I think the change makes sense for the worker flow only, but awkward for submission flow

Reply:
No, this code is not used for submitter, it's pure for worker/customer CLI flow.